client/src/styles/main.css

```diff
@@ -0,0 +1,3 @@
+.main {
+  width: 100vw;
```

Comment:
doesn't change anything on my side

Reply:
You are probably better off writing 100%